packages/@aws-cdk/aws-appmesh/test/integ.mesh.ts

```diff
@@ -17,6 +18,39 @@ const namespace = new cloudmap.PrivateDnsNamespace(stack, 'test-namespace', {
   name: 'domain.local',
 });
 
+const ca = new CfnCertificateAuthority(stack, 'CA', {
```

Comment:
Let's not spin up an actual CA for these tests. I feel as if spinning up other resources in our integ tests add a lot of extra time and complexity to our integ tests where all we are doing is accepting an ARN. We may want to validate with Adam but I have a feeling it is unnecessary

Reply:
changed to file